x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/action/EnrichCoordinatorProxyAction.java

```diff
@@ -57,11 +58,18 @@ private EnrichCoordinatorProxyAction() {
     public static class TransportAction extends HandledTransportAction<SearchRequest, SearchResponse> {
 
         private final Coordinator coordinator;
+        private final EnrichCache enrichCache;
```

Comment:
Are we making use of this anywhere in the proxy action?

Reply:
No, I initially did caching from here, but then moved it to the processor. The reason that in case something is cached then invoking coordinator action can be skipped as well.

Reply:
Fixed: 55e6e2424b7c680324dacf04a44e1b586220b72c